geoips/plugins/modules/algorithms/pmw_tb/pmw_37pct.py

```diff
@@ -10,19 +10,20 @@
 # # # for more details. If you did not receive the license, for more information see:
 # # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
 
-"""Data manipulation steps for "37pct" product.
+"""Passive Microwave 37 MHz Polarization Corrected Temperature.
 
-This algorithm expects Brightness Temperatures in units of degrees Kelvin
+Data manipulation steps for the "37pct" product.
+This algorithm expects Brightness Temperatures in units of degrees Kelvin.
 """
 import logging
 
 LOG = logging.getLogger(__name__)
 
 family = "list_numpy_to_numpy"
-description = "Passive Microwave 37 MHz Polarization Corrected Temperature"
+interface = "algorithms"
```

Comment:
@jsolbrig Please confirm this is CORRECT, before we update all the others.

Reply:
This look good to me. Just one nit-pick. I'd prefer to reorder the attributes a little bit:
```python
interface = "algorithms"
family = "list_numpy_to_numpy"
name = "pmw_37pct"
```